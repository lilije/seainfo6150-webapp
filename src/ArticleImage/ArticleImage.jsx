import React from "react";
import styles from "./ArticleImage.module.css";

const ArticleImage = ({ imageUrl, title }) => {

    return (
        <div>
            <img src={imageUrl} alt="image" />
            <h1>{title}</h1>
        </div>
    )
};

export default ArticleImage;