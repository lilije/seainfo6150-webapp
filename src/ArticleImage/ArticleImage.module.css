h1 { color: red; }

@media screen and (min-width: 800px) {
    img {
        margin-top: 50px;
        width: 300px;
        border-radius: 50%;
        border: 10px solid red;
    }
}