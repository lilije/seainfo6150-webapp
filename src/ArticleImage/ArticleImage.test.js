import React from "react";
import ArticleImage from "./ArticleImage";

describe("ArticleImage tests", () => {
  it("renders correctly", () => {
    let imageUrl = "https://cdn.cnn.com/cnnnext/dam/assets/181119170125-ploughgate-creamery-butter-ball-super-169.jpg";
    let articleTitle = "Butter is back, just in time for the holidays";
    
    const { container } = render(<ArticleImage imageUrl={imageUrl} articleTitle={articleTitle} />);
    expect(container).toMatchSnapshot();
  });
});
