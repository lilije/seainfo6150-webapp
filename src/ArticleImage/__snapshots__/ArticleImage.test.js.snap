// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleImage tests renders correctly 1`] = `
<div>
  <div>
    <img
      alt="image"
      src="https://cdn.cnn.com/cnnnext/dam/assets/181119170125-ploughgate-creamery-butter-ball-super-169.jpg"
    />
    <h1 />
  </div>
</div>
`;
